.travis.yml

```diff
@@ -4,7 +4,7 @@ services:
  - docker
 env:
   matrix:
-   - HHVM_VERSION=3.24-lts-latest
+   - HHVM_VERSION=3.26.3
```

Comment:
Actually just remove this line - there's two cases:
- it's equivalent to 'latest' below
- it's not, in which case, 3.26.3 is unsupported and you dion't want to be testing with it

You might want to also add 'nightly'